index.js

```diff
@@ -6,13 +6,15 @@ var balanced = require("balanced-match")
 var colorFn = require("css-color-function")
 var helpers = require("postcss-message-helpers")
 
+var colorFnRegExp = /(^|[^\w\-])color\(/;
+
 /**
  * PostCSS plugin to transform color()
  */
 module.exports = postcss.plugin("postcss-color-function", function() {
   return function(style) {
     style.eachDecl(function transformDecl(decl) {
-      if (!decl.value || decl.value.indexOf("color(") === -1) {
+      if (!decl.value || !colorFnRegExp.test(decl.value)) {
```

Comment:
Oh sorry. We missed something when communicating. We need here a stupid indexOf() test. Even if this might detect a `hover-color(`, this will be a lot faster since most props don't use `.*color(` function.
Regex is cool in transformColor in order to get an accurate result, but the stupid test should be fast. So please use the previous indexOf() here, even if it will miss some non real color() calls, it will be fast and the transformColor() function will do a better check.